src/main/telemetry/frsky.c

```diff
@@ -55,12 +55,11 @@ uint32_t frskyGetFlightMode(void)
     // ten thousands column
     if (FLIGHT_MODE(FLAPERON))
         tmpi += 10000;
-    if (FLIGHT_MODE(AUTO_TUNE))
-        tmpi += 20000;
     if (FLIGHT_MODE(FAILSAFE_MODE))
         tmpi += 40000;
-    // todo: prevent 16-bit variable from overflow, 
-    // D-series receivers transmit only 16-bit values
+    else
+        if (FLIGHT_MODE(AUTO_TUNE))
+            tmpi += 20000;
```

Comment:
Maybe a note to clarify why it's being done this way (as a year from now someone will probably see this as a mistake and "correct" it).  Something like this:

```
// intentionally reverse order and else if to prevent 16 bit overflow
```